r4e-aarch-image.sh

```diff
@@ -0,0 +1,80 @@
+# script for creating an image that points to a certain ostree commit from web server
```

Comment:
Do you think it would make sense to create common script with shared functions and source it here and in `https://github.com/ydayagi/r4e/blob/main/r4e-iso.sh`?

Reply:
That can be doable, but depends on the needs.
Having the same script will require additional parameters and the output will not be the same.

For x86 the output is ISO, since the boot image supports it.
However, for aarch64 there is no isolinux folder in its boot-iso, hence, it cannot be packaged the same way done for x86.
In composer of 8.5 there should be an option to save the image as live-iso which can be common for both.

Reply:
What is the  reason for not doing it now?

Reply:
Mainly time. Investing time in unifying the scripts for 8.4 while the target is 8.5 will consume time that we can spare and invest in 8.5 in which the composer allows saving the image as iso instead of using the current method which isn't supported for aarch.